### PR TITLE
feat: add allIdenticalTo

### PR DESCRIPTION
**Pull request by char0n**
Ref #525

### CODE REVIEW

File: src/allIdenticalTo.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * @func allIdenticalTo
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.11.0|v2.11.0}
+ * @category List
+ * @sig a -> [b] -> Boolean
+ * @param {*} val User provided value to check the `list` against
+ * @param {Array} list The list of values
+ * @return {boolean}
+ * @see {@link RA.allIdentical|allIdentical}
```
Comment (guillaumearm) [inline src/allIdenticalTo.js]: add a link to `R.identical` ?

File: test/allIdenticalTo.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+describe('allIdenticalTo', function() {
+  context('given all items are identical', function() {
+    specify('returns true', function() {
+      assert.isTrue(RA.allIdenticalTo(4, [4, 4, 4, 4]));
+    });
+  });
+
+  context('given all items are deeply equal', function() {
+    specify('should return true', function() {
```
Comment (guillaumearm) [inline test/allIdenticalTo.js]:
```suggestion
    specify('should return false', function() {
```

File: src/allIdenticalTo.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * @example
+ *
+ * RA.allIdenticalTo(1, [ 1, 2, 3, 4 ]); //=> false
+ * RA.allIdenticalTo(1, [ 1, 1, 1, 1 ]); //=> true
+ * RA.allIdenticalTo([]); //=> true
```
Comment (guillaumearm) [inline src/allIdenticalTo.js]:
```suggestion
 * RA.allIdenticalTo(1, []); //=> true
```

File: src/allIdenticalTo.js (hunk 1 above)
Comment (char0n) [reply src/allIdenticalTo.js]: yep; will add

**Comment by char0n** [comment]
Everything addressed. Thank you for review.

**Pull request merged**